src/main/java/seedu/address/model/property/Ask.java

```diff
@@ -0,0 +1,58 @@
+package seedu.address.model.property;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.commons.util.AppUtil.checkArgument;
+
+/**
+ * Represents ask price (minimum selling price) of property in the address book.
+ * Guarantees: details are present and not null, field values are validated, immutable.
+ */
+public class Ask {
+    public static final String MESSAGE_CONSTRAINTS =
+            "Ask price should be an integer";
+    public static final String VALIDATION_REGEX = "\\d+";
+    public final String value;
```

Comment:
should use private fields + getters?

Reply:
Hmm, I believe that the convention is that these fields are public in all the other classes in Model package. I'll leave this as a issue to resolve for all the classes in this package.